analytics/policy/services/business_licence_client.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import urllib3
+from models.address import Address
+from models.listing import Listing
+from policy.utils.dict_utils import merge_dicts
+from urllib3.exceptions import HTTPError
+
+
+class BusinessLicenceClient:
+    BASE_URL = "https://opendata.vancouver.ca/api/explore/v2.1/catalog/datasets/business-licences/records"
+
+    def __init__(self) -> None:
+        self.session = urllib3.PoolManager()
+
+    def filter_by_licence_number(self, licence_number: str) -> dict:
+        # Omit licence_number validity check since it should be checked during policy evaluation
+        return {"where": f'licencenumber="{licence_number}"'}
+
+    def filter_by_short_term_rental_business(self) -> dict:
+        return {"where": 'businesstype="Short-Term Rental"'}
+
+    def licence_status_query(self) -> dict:
```

Comment:
these are not public methods

Reply:
Agreed, will add underscore to indicate private methods